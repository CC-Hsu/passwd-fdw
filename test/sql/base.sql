\set ECHO none
BEGIN;
\i sql/passwd-fdw.sql
\set ECHO all

CREATE SERVER passwd_svr FOREIGN DATA WRAPPER passwd_fdw;

CREATE FOREIGN TABLE passwd(
   name   text,
   passwd text,
   uid    int,
   gid    int,
   gecos  text,
   dir    text,
   shell  text
) SERVER passwd_svr;

SELECT * FROM passwd;

ROLLBACK;
