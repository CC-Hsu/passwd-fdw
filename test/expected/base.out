\set ECHO 0
-- You should write your tests
SELECT passwd-fdw('test');
 passwd-fdw 
------------
 
(1 row)

ROLLBACK;
